srcpkgs/qperf/template

```diff
@@ -1,14 +1,16 @@
 # Template file for 'qperf'
 pkgname=qperf
 version=0.4.11
-revision=1
+revision=2
 build_style=gnu-configure
 hostmakedepends="automake"
+makedepends="rdma-core-devel libibmad-devel"
+depends="rdma-core libibmad"
 short_desc="Measure RDMA and TCP/UDP latency and throughput performance"
-maintainer="Rich G <rich@richgannon.net>"
+maintainer="Rich Gannon <rich@richgannon.net>"
 license="GPL-2.0-or-later"
 homepage="https://github.com/linux-rdma/qperf"
-distfiles="https://github.com/linux-rdma/qperf/archive/v${version}.tar.gz"
+distfiles="${homepage}/archive/v${version}.tar.gz"
```

Comment:
IMHO you should leave it as it was and not use `${homepage}` here not least for better readability.

Reply:
Fixed.